net/routing/Makefile

```diff
@@ -0,0 +1,7 @@
+PLUGIN_NAME=       routing
+PLUGIN_VERSION=    0.0.1
```

Comment:
0.1 is enough... we can go up to 0.9999999 if we want -- for pkg and release management this doesn't really matter as updates are updates.